CHANGELOG.md

```diff
@@ -1,3 +1,75 @@
+
+### google-maps
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  avoid runtime error on server ([8f36604](https://github.com/angular/components/commit/8f36604)) |
+| bug fix |  incorrect variable access for server-side rendering check ([#19166](https://github.com/angular/components/issues/19166)) ([b5864ca](https://github.com/angular/components/commit/b5864ca)) |
+
+### youtube-player
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  unable to bind to events after initialization ([#18996](https://github.com/angular/components/issues/18996)) ([0695e82](https://github.com/angular/components/commit/0695e82)) |
+
+### material-experimental
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **mdc-form-field:** add  filled MDC text field class ([#19103](https://github.com/angular/components/issues/19103)) ([54c5633](https://github.com/angular/components/commit/54c5633)) |
+| bug fix |  **mdc-progress-bar:** better server-side rendering support ([#19036](https://github.com/angular/components/issues/19036)) ([7a99cfa](https://github.com/angular/components/commit/7a99cfa)) |
+| bug fix |  **mdc-button:** strong focus indication always visible on fab buttons ([#19169](https://github.com/angular/components/issues/19169)) ([11786b7](https://github.com/angular/components/commit/11786b7)) |
+| bug fix |  **mdc-chips:** not disabling all animations when animations are disabled ([#18440](https://github.com/angular/components/issues/18440)) ([d25fe8b](https://github.com/angular/components/commit/d25fe8b)) |
+| bug fix |  **mdc-form-field:** missing styles for native select controls ([#19140](https://github.com/angular/components/issues/19140)) ([90c1474](https://github.com/angular/components/commit/90c1474)) |
+| bug fix |  **mdc-input:** avoid double event listeners in ivy ([#19052](https://github.com/angular/components/issues/19052)) ([2a4da91](https://github.com/angular/components/commit/2a4da91)) |
+| bug fix |  **mdc-radio:** avoid bundling styles from base radio button ([#19032](https://github.com/angular/components/issues/19032)) ([4557d58](https://github.com/angular/components/commit/4557d58)) |
+| bug fix |  **mdc-radio:** strong focus indicator not working ([#19091](https://github.com/angular/components/issues/19091)) ([b713104](https://github.com/angular/components/commit/b713104)) |
+| bug fix |  **mdc-slide-toggle:** strong focus indication not visible ([#19048](https://github.com/angular/components/issues/19048)) ([a507873](https://github.com/angular/components/commit/a507873)), closes [#18895](https://github.com/angular/components/issues/18895) |
+| bug fix |  **mdc-table:** not supporting multiple themes ([#18931](https://github.com/angular/components/issues/18931)) ([d564f6e](https://github.com/angular/components/commit/d564f6e)) |
+
+### autocomplete
```

Comment:
It's a little odd seeing `autocomplete` and `select` at the same level as package headers here.

Reply:
Good catch, I missed that. That's not supposed to happen.